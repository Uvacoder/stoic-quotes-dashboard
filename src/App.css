.App {
  text-align: center;
}

.App-logo {
  height: 40vmin;
}

.App-header {
  /* background-color: #282c34; */
  /* min-height: 100vh; */
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  /* background-image: url('https://bing.biturl.top/?resolution=1920&format=image&index=0&mkt=zh-CN'); */
  /* color: white; */
}

.App-link {
  color: #09d3ac;
}


