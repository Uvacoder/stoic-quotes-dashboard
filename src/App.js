import React from 'react';
import Clock from './Clock';
import Quote from './Quote';
import Background from './Background';
import Overlay from './Overlay';
import styled from 'styled-components';
import { Reset } from 'styled-reset'

const GlobalStyle = styled.div`
  text-shadow: 2px 3px 4px rgba(0,0,0,0.40);
  /* text-shadow: 0px 4px 8px rgba(0,0,0,0.77); */
`;

class App extends React.Component {
  render() {
    return(
      <GlobalStyle>
        <Overlay />
        <Reset />
        <Clock />
        <Quote />
        <Background />
      </GlobalStyle>
    ) 
  };
}

export default App;
