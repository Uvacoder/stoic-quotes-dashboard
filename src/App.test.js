import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import App from './App';

beforeAll(() => {
  // Lock Time
  jest.spyOn(Date, 'now').mockImplementation(() => 1487076708000);
});

afterAll(() => {
  // Unlock Time
  jest.spyOn(Date, 'now').mockRestore();
});


describe('<App />', () => {
  test('snapshot renders', () => {
    const component = renderer.create(<App />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
  it('renders dom structure without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<App />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
})

