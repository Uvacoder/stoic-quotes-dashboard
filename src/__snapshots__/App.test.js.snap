// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> snapshot renders 1`] = `
<div
  className="sc-bZQynM eKjPnv"
>
  <div
    className="sc-bwzfXH hfSMDM"
  >
    <div
      className="sc-bdVaJa yaRiT"
    >
      1:51:48 PM
    </div>
  </div>
  <div
    className="sc-htpNat knwwOj"
  >
    <div
      className="sc-bxivhb hpnUox"
    />
  </div>
  <div
    className="sc-ifAKCX ivIpuC"
  >
    <img
      alt=" "
      className="sc-EHOje kzWOdU"
      src={Array []}
    />
  </div>
</div>
`;
